tests/functional/disaster-recovery/regional-dr/test_failover_and_relocate_discovered_apps.py

```diff
@@ -28,27 +28,35 @@ class TestFailoverAndRelocateWithDiscoveredApps:
     """
 
     @pytest.mark.parametrize(
-        argnames=["primary_cluster_down", "pvc_interface"],
+        argnames=["primary_cluster_down", "pvc_interface", "kubeobject", "recipe"],
```

Comment:
Is it possible to have apps with kubeobject protection and recipe protection together ? If possible , that will improve coverage and the parameters kubeobject and recipe can be removed.

Reply:
This can be taken as enhancement https://github.com/red-hat-storage/ocs-ci/issues/11903